pyemail.py

```diff
@@ -93,10 +93,10 @@ def Conectar(self, servidor, usuario=None, clave=None, puerto=25):
                 self.smtp.starttls()
             if usuario and clave:
                 # convertir a string (hmac necesita string "bytes")
-                if isinstance(usuario, str):
-                    usuario = usuario.encode("utf8")
-                if isinstance(clave, str):
-                    clave = clave.encode("utf8")
+                # if isinstance(usuario, str):
```

Comment:
- [ ] Please leave this 2 if for python2 only

Reply:
- [x] Please leave this 2 if for python2 only

Reply:
made the required changes